R_scripts/tree_proximity.R

```diff
@@ -51,9 +63,14 @@ print(BA_cumdist)
 
 BA_dist <- ggplot(data = collar_to_tree_prox, aes( x = Distance_m, y = BA_sqm, group = Collar)) + 
   geom_point() + 
-  facet_grid(~ Collar) +
+  facet_wrap(~ Collar) +
   ggtitle("Distribution of basal area")
 print(BA_dist)
 
-#%>%
-#  left_join(licorDat, by = "Collar")
+# Calculate cumulative basal area at each distance 
+BA_dat <- list()
+for (i in 1:10) {
+  m <- collar_to_tree_prox[which(collar_to_tree_prox$Distance_m <= 5),]
```

Comment:
Shouldn't this be `<= i`?

Reply:
Yes, I think I was testing distances out